### DISCUSSION TITLE
[Feature] Show Recent "used" album

### DISCUSSION THREAD

**Discussion by xfaustus** [immich-app/immich#5203] [Feature Request]
### The feature

When adding photos to an album it would be great if recent albums list could be last/latests albums used to add photos, so if i'm searching photos to add in specific album i don't need to search again in list at each add

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can sort the albums page by 'last modified'.

**Reply by xfaustus** [reply]
Hello maybe i can explain better. Here are the steps to reproduce.
1. goto Photos
2. select one or more photos
3. click on Menu and select add to album
4. Note that album list doesn't follow sort preference.
